src/chat_manager.ts

```diff
@@ -68,11 +69,34 @@ export default class ChatManager {
   }
 
   connect(options: ConnectOptions) {
+    const cursorsReq: Promise<any> = this.userId
```

Comment:
It feels a bit opaque to have the functionality of the cursors dependent on having the `userId` property set. 

I think we'd need to make this more obviously configurable so that it's clear what the options are when it comes to providing the `userId` or not.

Reply:
Ahh yeah I did orinally have a "disable cursors" option, but decided it might have been overkill. The only reason there is a select on the userId being present is because it doesn't make sense to get cursors for the current user if we don't know who the current user is. This isn't really a behaviour change, it's more like "always get cursors unless this is impossible". Also, if we don't make this initial cursors request, it doesn't disable the setting and subscribing to cursors later on, it just means we start with a blank slate rather than starting out knowing all our initial cursors.

Reply:
edit: I suppose really I'm working under my biased assumption that basically `userId` should be a required property... another one to discuss tomorrow 😛